code/modules/mob/living/carbon/diona_base.dm

```diff
@@ -415,21 +415,22 @@ var/list/diona_banned_languages = list(
 //GETTER FUNCTIONS
 
 /mob/living/carbon/proc/get_lightlevel_diona(var/datum/dionastats/DS)
-	var/light_factor = 1
+	var/light_factor = 1.15
```

Comment:
diona light intake levels have been buffed a bit. All the lights were balanced around this new multiplier